binancial/data/get_trades_realtime.py

```
@@ -1,45 +1,95 @@
+            logger.info(f"Starting trade stream for {symbol}")
+            stream.start()
+            
+            # Store socket id for proper cleanup
+            socket_id = stream.start_trade_socket(
+                callback=handle_socket_message,
+                symbol=symbol
+            )
+            
+            stream.join()
+            
+    except IOError as io_error:
```

Ruff (0.8.2)

Replace aliased errors with `OSError`

Replace `IOError` with builtin `OSError`

(UP024)